src/main/kotlin/com/terraformation/backend/tracking/db/PlantingSiteStore.kt

```diff
@@ -403,6 +404,26 @@ class PlantingSiteStore(
     }
   }
 
+  fun deletePlantingSite(plantingSiteId: PlantingSiteId) {
+    requirePermissions { deletePlantingSite(plantingSiteId) }
+
+    // Inform the system that we're about to delete the planting site and that any external
+    // resources tied to it should be cleaned up.
+    //
+    // This is not wrapped in a transaction because listeners are expected to delete external
+    // resources and then update the database to remove the references to them; if that happened
+    // inside an enclosing transaction, then a listener throwing an exception could cause the system
+    // to roll back the updates that recorded the successful removal of external resources by an
+    // earlier one.
+    //
+    // There's an unavoidable tradeoff here: if a listener fails, the planting site data will end up
```

Comment:
Would these failures show up in Data Dog?

Reply:
Yes, the exception would bubble up and be logged.